libraries/state_history/include/eosio/state_history/serialization.hpp

```diff
@@ -359,13 +359,15 @@ ST& operator<<(ST& ds, const history_serial_wrapper<eosio::chain::chain_config>&
 
 template <typename ST>
 ST& operator<<(ST& ds, const history_serial_wrapper<eosio::chain::global_property_object>& obj) {
-   fc::raw::pack(ds, fc::unsigned_int(1));
+   fc::raw::pack(ds, fc::unsigned_int(2)); // for global_property_v2
```

Comment:
const fc::unsigned_int global_property_version = 2;
fc::raw::pack(ds, global_property_version);

I think would make this a little clearer